packages/volto/src/components/manage/Contents/ContentsDeleteModal.jsx

```diff
@@ -98,11 +228,24 @@ const ContentsDeleteModal = (props) => {
           target: result,
         })),
       );
-      const source_by_uid = breaches.reduce(
+      // Filter out breaches where breach.source is to be deleted
+      const filteredBreaches = breaches.filter(
+        (breach) =>
+          !itemsToDelete.some((item) => breach.source['@id'].endsWith(item)),
+      );
+      // If no breaches are found, return early
+      if (filteredBreaches.length === 0) {
+        setContainedItemsToDelete([]);
```

Comment:
```suggestion
        setContainedItemsToDelete(0);
```

Consider initializing containedItemsToDelete as a number instead of an array for consistency.